requirements/base.txt

```diff
@@ -1,21 +1,81 @@
-boto==2.42.0                        # MIT
-Django==1.11.5						# BSD License
-django-countries==4.5               # MIT
-djangorestframework==3.6.2			# BSD
-django-rest-swagger==2.1.2  		# BSD
-djangorestframework-csv==2.0.0		# BSD
-django-storages==1.5.2              # BSD
-elasticsearch-dsl==0.0.11           # Apache 2.0
-ordered-set==2.0.2                  # MIT
-tqdm==4.11.2                        # MIT
-urllib3==1.21.1                     # MIT
-
-# markdown is used by swagger for rendering the api docs
-Markdown==2.6.6    					# BSD
-
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    make upgrade
```

Comment:
Did you manually edit the file re-generation instructions?  In other repos we're calling a shell script at the end of `make upgrade` to do that, but I didn't see one here.

Reply:
Yes, I manually edit instructions. Just saw shell script written for edx-platform. Do we just need to add "re-generation of instructions" part? Is there any other thing we should be looking for to add in shell script?

Reply:
For this repo, that's probably all it needs to do.  There's a version in xqueue that does just that: https://github.com/edx/xqueue/blob/master/script/post-pip-compile.sh .  The version you added here should be fine also.

Reply:
xqueue version of `post-pip-compile.sh` script looks better. I have updated the script correspondingly.